src/atoms/View/__tests__/View.web.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { renderWithTheme } from '../../../_helpers/testing-web';
+
+import View from '../index';
+
+describe('Renders <View /> correctly', () => {
```

Comment:
`Renders <View />` no need of adding correctly since this is a test suite and not a test.